Sources/Datadog/DatadogInternal/DatadogCoreProtocol.swift

```diff
@@ -46,6 +46,22 @@ public protocol DatadogCoreProtocol {
 }
 
 extension DatadogCoreProtocol {
+    /// Registers a feature instance by its type description.
+    ///
+    /// - Parameter instance: The feaure instance to register
+    func register<T>(feature instance: T?) {
+        registerFeature(named: String(describing: T.self), instance: instance)
+    }
```

Comment:
Features' name in v1 does not make much sense, I'm thinking of using this method for all v1 features!